src/udev/udev-rules.c

```diff
@@ -2441,8 +2440,8 @@ static int udev_rule_apply_parent_token_to_event(
                         if (r == 0)
                                 break;
                 }
-                if (!line->current_token)
-                        /* All parent tokens match. But no assign tokens in the line. Hmm... */
+                if (r > 0)
```

Comment:
Could `r` be uninitialized here, for example if the list is empty?

Reply:
The list is not empty. I will add an assertion soon.